src/rest_framework_jwt/compat.py

```diff
@@ -2,8 +2,30 @@
 
 from __future__ import unicode_literals
 
+from datetime import datetime
+
+from django import VERSION
+
+from .settings import api_settings
 
 try:
     from django.urls import include, url
 except ImportError:
     from django.conf.urls import include, url  # noqa: F401
+
+def has_set_cookie_samesite():
+    VERSION >= (2,1,0)
```

Comment:
Isn't this line missing a `return` statement?

Reply:
oops. You just caught a python newbie :)